Formula/carl-storm.rb

```diff
@@ -0,0 +1,39 @@
+class CarlStorm < Formula
+  desc "Computer ARithmetic and Logic library for the probabilistic model checker Storm"
+  homepage "https://ths-rwth.github.io/carl/"
+  url "https://github.com/moves-rwth/carl-storm/archive/refs/tags/14.25.tar.gz"
+  version "14.25"
+  sha256 "511740d53c2a6a41c3ccb3bd3b2d9ec89d0576f37b8804fc15fbe083e7a357da"
+
+  head "https://github.com/moves-rwth/carl-storm.git", using: :git
+
+  option "with-thread-safe", "Build with thread-safe support"
+
+  depends_on "boost"
+  depends_on "cmake"
+  depends_on "eigen"
+  depends_on "gmp"
+  on_intel do
+    depends_on "cln" => :optional
+    depends_on "ginac" => :optional
+  end
+  depends_on "moves-rwth/misc/cocoalib" => :optional
```

Comment:
Do we need this for Storm? As far as I am aware we could remove this dependency.